packages/data/src/onboarding/actions.js

```diff
@@ -47,6 +47,108 @@ export function setProfileItems( profileItems, replace = false ) {
 	};
 }
 
+export function getTaskListsError( error ) {
+	return {
+		type: TYPES.GET_TASK_LISTS_ERROR,
+		error,
+	};
+}
+
+export function getTaskListsSuccess( taskLists ) {
+	return {
+		type: TYPES.GET_TASK_LISTS_SUCCESS,
+		taskLists,
+	};
+}
+
+export function snoozeTaskError( taskId, error ) {
+	return {
+		type: TYPES.SNOOZE_TASK_ERROR,
+		taskId,
+		error,
+	};
+}
+
+export function snoozeTaskRequest( taskId ) {
+	return {
+		type: TYPES.SNOOZE_TASK_REQUEST,
+		taskId,
+	};
+}
+
+export function snoozeTaskSuccess( taskId ) {
+	return {
+		type: TYPES.SNOOZE_TASK_SUCCESS,
+		taskId,
```

Comment:
I think we should pass in the entire `task` here, as the `task` includes `snoozedUntil`, which we probably want to set in our redux state as well.